game/hud/src/widgets/Warband/services/session/layout.ts

```diff
@@ -12,6 +12,9 @@ const SET_POSITION = 'cu-ui/hud/layout/SET_POSITION';
 const SAVE_POSITION = 'cu-ui/hud/layout/SAVE_POSITION';
```

Comment:
Looks like this file has been copied and is slowly diverging, we should make a common version and specialize where needed